source/docs/05-mensural.xml

```diff
@@ -165,33 +165,33 @@
                   </list>
                   <p>The last three values are to be used exclusively in <hi rend="italic">Ars antiqua</hi> mensural notation, where <val>maior</val> and <val>minor</val> refer to types of <hi rend="italic">semibreves</hi>, and <val>duplex</val> refers to a type of <hi rend="italic">longa</hi>. Examples of each of these six values are presented below. In these examples, the ‘voice’ staff renders the notes in the code snippet, and the ‘reference’ staff, together with the dotted bar lines, are shown to help to visualize the relative values of the notes in the ‘voice’ staff.</p>
                   <p>The following example illustrates an <hi rend="italic">alteration</hi> (the second <hi rend="italic">brevis</hi>) in <hi rend="italic">modus minor perfectus</hi>. Notice that the second <hi rend="italic">brevis</hi> has doubled its regular value, it has been altered, unlike the first one. <figure>
-                        <head/>
+                        <head>Example on alteration: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
```

Comment:
```suggestion
                        <head>Example on alteration (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```

Reply:
I suggest:
<head>Alteration: (The bottom staff, together with the dotted barlines, is used to help visualize the durational values of the notes in the upper staff.)</head>